crates/router/src/core/errors/utils.rs

```diff
@@ -43,111 +43,119 @@ impl<T> StorageErrorExt<T, errors::ApiErrorResponse>
     }
 }
 
-pub trait ConnectorErrorExt {
+pub trait ConnectorErrorExt<T> {
```

Comment:
Do you think naming this `ConnectorResultExt` makes more sense, since we're implementing the trait on `Result`?

Reply:
Since we don't have a type `ConnectorResult`  and the trait doesn't do anything with the Ok part . I think It makes sense to keep it as it is.